### ISSUE TITLE
Bartender's little coin icon won't go away in the new update, even after checking the shop

### ISSUE THREAD

**Issue by thekirbykid20067** [Foohy/jazztronauts#145]
As the title says.
![20230506031402_1](https://user-images.githubusercontent.com/56051350/236609308-7e9fa040-04e2-412e-a40e-335ee533f99d.jpg)
![20230506031405_1](https://user-images.githubusercontent.com/56051350/236609310-2514a7c8-99b1-4dfd-b75e-cb80ea0e1dc2.jpg)
![20230506031406_1](https://user-images.githubusercontent.com/56051350/236609312-83c24f2d-e62c-4362-8bfa-047585fc0fef.jpg)
![20230506031407_1](https://user-images.githubusercontent.com/56051350/236609313-30cac42c-4744-4fc5-8ded-d3acd8def66d.jpg)

**Comment by Foohy** [comment]
Good catch, should be fixed in ae47016ed36f322b91967954b2d4c4274e7d6a09
